libs/sdk-ui-vis-commons/package.json

```diff
@@ -88,6 +89,8 @@
         "@types/react-dom": "^16.9.3",
         "@types/react-measure": "2.0.5",
         "@types/react": "^16.9.11",
+        "@types/stringify-object": "^3.2.0",
+        "@types/uuid": "^8.3.0",
```

Comment:
We dont need this reference anymore generating of random id is moved outside of this package